cmd/message_input.go

```diff
@@ -18,12 +18,13 @@ import (
 type MessageInput struct {
 	*tview.TextArea
 	replyMessageIdx int
+	replyMessageID discord.MessageID 
 }
 
 func newMessageInput() *MessageInput {
 	mi := &MessageInput{
 		TextArea:        tview.NewTextArea(),
-		replyMessageIdx: -1,
+		replyMessageID: 0,
```

Comment:
Remove this line as Go already initializes unspecified fields of a `struct` as their zero value.